build-conflicts.php

```diff
@@ -34,6 +34,11 @@
 use Roave\SecurityAdvisories\AdvisorySources\GetAdvisoriesFromGithubApi;
 use Roave\SecurityAdvisories\AdvisorySources\GetAdvisoriesFromMultipleSources;
 
+use function array_keys;
+use function count;
+use function file_get_contents;
```

Comment:
```suggestion
use Psl\Filesystem;
```

$content = Filesystem\read_file($filename);

Reply:
thank you!